src/stories/Forms/Button/index.tsx

```diff
@@ -83,7 +83,7 @@ export interface ButtonProps {
   color?: 'black' | 'lavender' | 'intenseOrange';
   disabled?: boolean;
   loading?: boolean;
-  onClick?: () => void;
+  onClick?: (e?: React.MouseEvent<HTMLButtonElement>) => void;
```

Comment:
I feel like we rarely specify the type of event in function props. If this is added, please account for touch events too like so: 
```javascript
(e?: React.TouchEvent<HTMLButtonElement> | React.MouseEvent<HTMLButtonElement>) => void;
```